Amplify/DefaultPlugins/AWSUnifiedLoggingPlugin/AWSUnifiedLoggingPlugin.swift

```diff
@@ -99,3 +99,11 @@ extension AWSUnifiedLoggingPlugin {
         return wrapper
     }
 }
+
+extension AWSUnifiedLoggingPlugin: AmplifyVersionable {
+    public var version: String {
+        let bundle = Bundle(for: type(of: self))
+        let version = bundle.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String
+        return version ?? "Not Available"
```

Comment:
Can we give default behavior for this protocol, instead of repeating the same code?

Reply:
When I originally thought about that I was stymied by `type(of:)` expecting a class, and `AmplifyVersionable` not being constrained to classes. But now that I'm thinking about it, I could write a conditional extension. Derp. Will fix.